src/SonarScanner.Shim/Resources.resx

```diff
@@ -152,7 +152,8 @@
 Possible causes:
   1. The project has not been built - the project must be built in between the begin and end steps
   2. An unsupported version of MSBuild has been used to build the project. Currently MSBuild 14.0 upwards are supported
-  3. The begin, build or end steps have not all been launched from the same folder</value>
+  3. The begin, build or end steps have not all been launched from the same folder
+  4. None of the analyzed projects have a GUID and you haven't used any solution (.sln)</value>
```

Comment:
I decided to add this 4th option in case the user build .Net Core/Std projects without a solution so the analysis will fail without any project analyzed and even if the log will now contain a message I think it is useful to have it here too.

Reply:
Minor suggested rewording: "... have a valid ProjectGuid and you have not used a solution (.sln)"

Not changed in this commit, but (3) should say "The begin, build **and** end steps..."